schema/pil.js

```diff
@@ -50,7 +118,7 @@ class PIL extends BaseModel {
           to: 'establishments.id'
         }
       },
-      transfers: {
+      pilTransfers: {
```

Comment:
Rename this to match the name of the joined table because otherwise the `softDeleted` queries through an error on the subqueries because they include clauses based on table name and do not respect any local aliases. This makes sure that the queries match the table name.